### PR TITLE
Ticket-which-a-user-has-control-over-list endpoint

### LINKED ISSUE #170: Create an endpoint that lists tickets you have control over
tbd

### PR DESCRIPTION
**Pull request by bvanderdrift**
Fixes #170

### CODE REVIEW

File: src/main/java/ch/wisv/areafiftylan/controller/TicketRestController.java [hunk 1]
```diff
@@ -42,4 +52,22 @@ public TicketTransferRestController(TicketService ticketService) {
...
+     *
+     * @return A collection of all TicketTypes and their availability
+     */
+    @RequestMapping(value = "/tickets/available", method = RequestMethod.GET)
+    public Collection<TicketInformationResponse> getAvailableTickets() {
+        return orderService.getAvailableTickets();
+    }
+
+    @PreAuthorize("isAuthenticated()")
+    @RequestMapping(value = "/tickets/incontrol", method = RequestMethod.GET)
```
Comment (skamoen) [inline src/main/java/ch/wisv/areafiftylan/controller/TicketRestController.java]: I'm not sure I like the "incontrol" part. You're not really in control over the tickets, just the seat placement. Let's come up with something better

File: src/main/java/ch/wisv/areafiftylan/service/TicketService.java [hunk 2]
```diff
@@ -58,4 +58,12 @@
...
      */
     void cancelTicketTransfer(String token);
+
+    /**
+     * Get all tickets over which a user has control. His own, as well as his teammates their tickets if the user is captain.
+     *
+     * @param user The user whose tickets are requested
+     * @return All tickets over which a user has control.
+     */
+    Collection<Ticket> getTicketsInControl(User user);
```
Comment (skamoen) [inline src/main/java/ch/wisv/areafiftylan/service/TicketService.java]: Same here

File: src/main/java/ch/wisv/areafiftylan/service/TicketServiceImpl.java [hunk 3]
```diff
@@ -132,4 +139,26 @@ private TicketTransferToken getTicketTransferTokenIfValid(String token){
 
         return ttt;
     }
+
+    @Override
+    public Collection<Ticket> getTicketsInControl(User u){
+        Collection<Ticket> ownedTickets = ticketRepository.findAllByOwnerUsername(u.getUsername());
+        Collection<Ticket> captainedTickets = getCaptainedTickets(u);
```
Comment (skamoen) [inline src/main/java/ch/wisv/areafiftylan/service/TicketServiceImpl.java]: I would change the name of this to `getTicketsFromTeamMembers()`

**Comment by skamoen** [comment]
The logic looks good, I just have some reservations about the naming part.

**Comment by bvanderdrift** [comment]
Fixed. Back to @skamoen

**Comment by skamoen** [comment]
LGTM

**Pull request merged**